modules/YamlStore/manifest.yaml

```diff
@@ -0,0 +1,13 @@
+description: Provides a (read-only) key-value store interface, backed by a simple YAML file.
```

Comment:
Maybe this can be reflected in the module name itself. So calling it `ReadOnlyYamlStore`, but I'm not 100% sure about this